TCAT/Controllers/RouteDetail+DrawerViewController.swift

```diff
@@ -59,14 +59,14 @@ class RouteDetailDrawerViewController: UIViewController {
     private var busDelayNetworkTimer: Timer?
     private let chevronFlipDurationTime = 0.25
     private let networking: Networking = URLSession.shared.request
-    private var route: Route!
+    let route: Route
```

Comment:
why are we making this non-private?

Reply:
I used `route` in `RouteDetailDrawerViewController`'s extension in another file

Reply:
made it back to private!